packages/@aws-cdk/aws-codepipeline/test/integ.pipeline-alexa-deploy.ts

```diff
@@ -0,0 +1,39 @@
+import alexa = require('@aws-cdk/alexa-ask');
+import s3 = require('@aws-cdk/aws-s3');
+import cdk = require('@aws-cdk/cdk');
+import codepipeline = require('../lib');
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-codepipeline-alexa-deploy');
+
+const pipeline = new codepipeline.Pipeline(stack, 'Pipeline');
+
+const sourceStage = new codepipeline.Stage(pipeline, 'Source', { pipeline });
+const bucket = new s3.Bucket(stack, 'PipelineBucket', {
+  versioned: true,
+  removalPolicy: cdk.RemovalPolicy.Destroy,
+});
+const sourceAction = new s3.PipelineSourceAction(stack, 'Source', {
+  stage: sourceStage,
+  outputArtifactName: 'SourceArtifact',
+  bucket,
+  bucketKey: 'key',
+});
+
+const deployStage = new codepipeline.Stage(pipeline, 'Deploy', { pipeline });
+
+const clientId = new cdk.SecretParameter(stack, 'AlexaClientId', {ssmParameter: '/Alexa/ClientId'});
```

Comment:
That integration test won't deploy unless the SSM Parameters exist, right? It'd be nice to somehow swap those out for phony values or locally-created entities instead, so the test can run without any pre-requisite set-up.

Reply:
Good point. I changed it.